src/software/thunderscope/robot_diagnostics/chicker.py

```diff
@@ -1,12 +1,22 @@
 import pyqtgraph as pg
-from pyqtgraph.Qt.QtCore import Qt
+from pyqtgraph.Qt.QtCore import *
 from pyqtgraph.Qt.QtWidgets import *
 from software.py_constants import *
 from proto.import_all_protos import *
+from enum import Enum
 import software.thunderscope.common.common_widgets as common_widgets
 
 from software.thunderscope.thread_safe_buffer import ThreadSafeBuffer
 
+MILLISECONDS_PER_SECOND = 1000
```

Comment:
We actually define some of the general constants in `constants.h` and use PyBind to also have access to them in Python. So similar to this, you don't have to redefine the constant. 
https://github.com/UBC-Thunderbots/Software/blob/97acde5458f2d79b36d99f7143139a3ea6a93041/src/software/simulated_tests/simulated_test_fixture.py#L21

Reply:
done